drivers/dose/dose.c

```diff
@@ -74,8 +74,10 @@ static dose_signal_t state_transit_blocked(dose_t *ctx, dose_signal_t signal)
         ctx->netdev.event_callback((netdev_t *) ctx, NETDEV_EVENT_ISR);
     }
 
-    /* Enable GPIO interrupt for start bit sensing */
-    gpio_irq_enable(ctx->sense_pin);
+    if (ctx->sense_pin != UART_UNDEF) {
```

Comment:
Here and through the rest of the file, I guess you mean `GPIO_UNDEF`?

Reply:
Yes, you are correct. Of course. I don't know how this mistake slipped in. I'm going to push fix-up commits. Thank you for having a look into this PR!